crates/gateway-config/src/subscriptions_protocol.rs

```diff
@@ -0,0 +1,38 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, PartialEq, Eq, Clone, Serialize)]
+#[serde(rename_all = "snake_case")]
+pub enum SubscriptionsProtocol {
+    ServerSentEvents,
+    Websockets,
```

Comment:
Shouldn't be plural IMHO

Reply:
Will change.

Reply:
for `wesocket` I mean, `ServerSentEvents` is indeed plural from what I saw